frontend/packages/core/src/Table/metadata-table.tsx

```diff
@@ -114,18 +116,22 @@ const MutableRow: React.FC<MutableRowProps> = ({ data, onUpdate, onReturn, valid
     <TableRow key={data.id}>
       <KeyCell data={data} />
       <TableCell>
-        <TextField
-          id={data.id}
-          name={data.name}
-          defaultValue={data.value}
-          type={data?.input?.type}
-          onChange={onUpdate}
-          onReturn={onReturn}
-          onFocus={onUpdate}
-          inputRef={validation.register}
-          helperText={error?.message || ""}
-          error={!!error || false}
-        />
+        <Grid container alignItems="center" wrap="nowrap" spacing={2}>
+          <TextField disabled id={data.id} name={data.name} defaultValue={data.value} />
```

Comment:
the text field widths are larger than figma design. i can override by giving them fix widths but unsure if that's best practice

Reply:
Let's override these for now to match the designs